services/app-api/package.json

```diff
@@ -16,15 +17,19 @@
     "url": "https://github.com/AnomalyInnovations/serverless-nodejs-starter.git"
   },
   "devDependencies": {
+    "@faker-js/faker": "^8.4.1",
     "@types/dompurify": "^2.3.4",
     "@types/jest": "^27.4.0",
     "@types/jsdom": "^20.0.0",
     "@types/jwt-encode": "^1.0.0",
     "@types/md5": "^2.3.2",
     "@types/object-hash": "^3.0.2",
+    "amazon-user-pool-srp-client": "^1.0.4",
     "aws-lambda": "^1.0.7",
     "aws-sdk-client-mock": "^4.0.0",
+    "axios": "^1.7.2",
```

Comment:
discussion: any particular reason you chose axios here? I think in some other apps we removed it in favor of the node native `fetch`, to reduce dependencies and resolve some security issues with axios. Those issues may now be fine in the latest, just seeing if it's an option. Here is an example of using [fetch in CARTS](https://github.com/Enterprise-CMCS/macpro-mdct-carts/blob/main/services/app-api/handlers/printing/printPdf.ts#L55-L70). I think the logic would be very similar.

Reply:
It thought axios was already a dependency of another library we were using, so I explicitly included it and thought no harm in using it. But looks I was incorrect. I can convert to using fetch!